### ISSUE TITLE
Stale sessions are not being reaped by the server

### ISSUE THREAD

**Issue by mbyczkowski** [superfly/wormhole#34]
We've had a number of reports that when a client is terminated, the server still shows old sessions as active. No actual repro, yet.

**Comment by doodles526** [comment]
@mbyczkowski Are these specifically reffing the SSH session type?

**Comment by mbyczkowski** [comment]
@doodles526 yes, that's what I have noticed on wormhole servers under load with multiple clients (and each with multiple connections/streams) opened. Hopefully, I'll be able to provide more info. Should figure out how to simulate this better.
